Tests/AWSSDKSwiftTests/Services/SSM/SSMTests.swift

```diff
@@ -0,0 +1,79 @@
+//
+// SSMTests.swift
+// written by Adam Fowler
+// SSM tests
+//
+import XCTest
+import Foundation
+@testable import AWSSDKSwiftCore
+@testable import SSM
+
+enum SSMTestsError : Error {
+    case noTopicArn
+}
+
+// testing json service
+
+class SSMTests: XCTestCase {
+
+    let client = SSM(
+            accessKeyId: "key",
+            secretAccessKey: "secret",
+            region: .apnortheast1,
+            endpoint: "http://localhost:4583",
```

Comment:
Do we maybe want to expose the host and port as `SSM_HOST` and `SSM_PORT` with fallbacks to `localhost` and `4583`, which would ensure local testability?

That would be great for my pr.

Would apply to all services.

Reply:
Do you mean setup an environment variable for each service? Might as well combine the host and the port.

Reply:
> Do you mean setup an environment variable for each service?
Yes since, we have different ports and so on. And maybe one want's to test locally against dynamo-local. Whatever.

> Might as well combine the host and the port.
That's true. Go ahead with that if you like!